packages/office-addin-debugging/src/start.ts

```diff
@@ -281,6 +281,12 @@ export async function startDebugging(manifestPath: string, appType: AppType, app
             }
         }
 
+        // enable Outlook sideloading
+        if (isDesktopAppType && app == OfficeApp.Outlook) {
```

Comment:
Could this not be integrated more into the sideloadOfficeAddIn() call below, so if the app is Outlook, it then does the different stuff? We really don't neeed to the console output to appear differently.  Does the "Enabled sideloading for Outlook" message mean much?

Reply:
sure we can do that - we really don't need the logging